src/librustc/session/config.rs

```diff
@@ -253,6 +253,7 @@ top_level_options!(
         // Include the debug_assertions flag into dependency tracking, since it
         // can influence whether overflow checks are done or not.
         debug_assertions: bool [TRACKED],
+        debug_prefix_map: Option<(String, String)> [TRACKED],
```

Comment:
This only allows a single remapping. It needs to allow an arbitrary number of remappings to be useful.